modules/ROOT/pages/branded-emails.adoc

```diff
@@ -0,0 +1,66 @@
+. Set *Profiles* to `System Administrator`.
+. Click *Save*.
+. Open the confirmation email sent to the notification sender email address and click the link to confirm the address.
+. Navigate to *Setup* and use *Quick Find* to go to *Custom Metadata Types*.
+. Click *ACM Community Config Metadata* to add a record.
+. Set *Label* to your API community's name.
+. Set *ACM Community Config Metadata Name* to your API community's name.
+. Open the *Developer Console* and open the *Query Editor*.
+. Enter `SELECT Id, Name FROM Network` and click *Execute*.
+. In the list of IDs and communities, choose the row with *Name* equal to your community's name, double click the ID, and copy it.
+. In the *ACM Community Config Metadata* field *NetworkId*, paste the ID.
+. In the *Query Editor*, enter `SELECT Id, Name FROM EmailTemplate` and click *Execute*.
+. In the list of IDs and email templates, choose the row with *Name* equal to *API Community Manager: Contract Approved*, double click the ID, and copy it.
+. In the *ACM Community Config Metadata* field *ContractApprovedEmailTemplateId*, paste the ID.
+. In the list of IDs and email templates, choose the row with *Name* equal to *API Community Manager: Contract Revoked*, double click the ID, and copy it.
+. In the *ACM Community Config Metadata* field *ContractRevokedEmailTemplateId*, paste the ID.
+. In the *Query Editor*, enter `SELECT Id, Address FROM OrgWideEmailAddress` and click *Execute*.
+. In the list of IDs and email addresses, choose the row with *Address* equal to the organization wide email address you set, double click the ID, and copy it.
+. In the *ACM Community Config Metadata* field *OrgWideEmailAddressId*, paste the ID.
+. Click *Save*.
+. Repeat this process for all additional communities, creating one record for each community.
++
+*NetworkId* is different for each community. *ContractApprovedEmailTemplateId*, *ContractRevokedEmailTemplateId*, and *OrgWideEmailAddressId* may be the same for multiple communities.
+. Navigate to *Setup*, use *Quick Find* to go to *Classic Email Templates*, and navigate to *Folder* > *ACM*.
+. Customize the *API Community Manager: Contract Approved* and *API Community Manager: Contract Revoked* templates, or create your own templates and configure the *ACM Community Config Metadata* to use them.
+. Configure an email relay as described in the Salesforce documentation page https://help.salesforce.com/articleView?id=emailadmin_email_relay_overview.htm&type=5[Send Email Through Email Relay, role=external, window=blank].
++
+If the relay is not configured, notification emails are sent from a Salesforce domain with a sender email address in your organization's domain. If the actual sender domain and sender email address domain do not match, many email providers mark the notification emails as spam.
+. Modify the headers of the external data source so the default notification emails with Anypoint branding are no longer sent.
```

Comment:
this should also be a section as it involves several steps "Disable Anypoint default email notifications" (with the previous comment, this will become the second to last section - before the new section "Complete permissions configuration in anypoint" which will be the last section). it could say something like this:

You can alter the email addresses of a member that is provided to Anypoint when the corresponding Anypoint user record is created through the impersonation mechanism so that the Anypoint user record email is not a valid address. This prevents the community member from receiving the standard Anypoint notification emails. 

To to this: 
1. go to salesforce setup > external data sources and click on the NAME of the Exchange external data source to open it in view mode
2. Scroll to the "Custom HTTP Headers" section and click Edit on "X-Email" header
3. add a suffix to the "Header field value" field, for example:  `$User.Email` + ".donotsend"`

Reply:
Done. Thank you!